### PR TITLE
Bulgarian Translation 12

### PR DESCRIPTION
**Pull request by Effektus**
76% translated

### CODE REVIEW

File: files/lang/bg.po [hunk 1]
```diff
@@ -8990,64 +9017,67 @@ msgid "game: offer to continue the game after victory condition"
...
+msgstr "Непозната сила възкръсва..."
 
 msgid ""
 "Check the newest version of the game at\n"
 "https://github.com/ihhub/\n"
 "fheroes2/releases"
 msgstr ""
+"Проверете най-новата версия на играта на\n"
+"https://github.com/ihhub/\n"
+"fheroes2/издания"
```
Comment (ihhub) [inline files/lang/bg.po:9080]: Please do not translate a part of web URL.

**Review by ihhub — CHANGES_REQUESTED** [review]

**Review by zenseii — CHANGES_REQUESTED** [review]
Hi, @Effektus.

I left a couple of comments here. Please have a look at it when you can.

File: files/lang/bg.po [hunk 2]
```diff
@@ -2664,7 +2666,7 @@ msgid "Audio"
 msgstr "Звукови"
 
 msgid "Configure"
-msgstr ""
+msgstr "Конфигуриране"
```
[the remark this answers is not in the document]
Comment (zenseii) [reply files/lang/bg.po:2669]: Of course you're free to decide what way to make this fit. You can also shorten the Text mode string below.

**Comment by zenseii** [comment]
Hi @Effektus, could you please solve the merge conflicts?

File: files/lang/bg.po [hunk 3]
```diff
@@ -7193,127 +7216,127 @@ msgid "Random Monsters 4"
...
-msgstr ""
+msgstr "Имунитет към огнени магии"
 
 msgid "Immune to Cold spells"
-msgstr ""
+msgstr "Имунитет към ледени магии"
 
 msgid "Immune to "
-msgstr ""
+msgstr "Имунитет към"
```
Comment (ihhub) [inline files/lang/bg.po:7246]: Please add a space at the end of the line.

File: files/lang/bg.po [hunk 4]
```diff
@@ -7193,127 +7216,127 @@ msgid "Random Monsters 4"
...
-msgstr ""
+msgstr "200% щети от студени магии"
 
 msgid "% damage from %{spell} spell"
-msgstr ""
+msgstr "% щети от %{spell} заклинание"
 
 msgid "% immunity to "
-msgstr ""
+msgstr "% имунитет към"
```
Comment (ihhub) [inline files/lang/bg.po:7330]: Also please add a space at the end of this line.

File: files/lang/bg.po [hunk 5]
```diff
@@ -7193,127 +7216,127 @@ msgid "Random Monsters 4"
...
 
 msgid "Lightning"
-msgstr ""
+msgstr "Светкавица"
 
 msgid "% damage from "
-msgstr ""
+msgstr "% щети от"
```
Comment (ihhub) [inline files/lang/bg.po:7336]: Please add a space at the end of the line.

File: files/lang/bg.po [hunk 6]
```diff
@@ -8993,64 +9020,67 @@ msgid "game: offer to continue the game after victory condition"
...
 msgid ""
 "Check the newest version of the game at\n"
 "https://github.com/ihhub/\n"
 "fheroes2/releases"
 msgstr ""
+"Проверете най-новата версия на играта на\n"
+"https://github.com/ihhub/\n"
+"fheroes2/releases"
 
 #~ msgid "Moved %{monster}"
```
Comment (ihhub) [inline files/lang/bg.po:9085]: Please remove all lines starting from this one till the end of the file since they are not in use anymore.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Effektus , I put several comments here. Could you please take a look at them?

File: files/lang/bg.po [hunk 7]
```diff
@@ -8993,64 +9020,64 @@ msgid "game: offer to continue the game after victory condition"
...
 "Check the newest version of the game at\n"
 "https://github.com/ihhub/\n"
 "fheroes2/releases"
 msgstr ""
-
-#~ msgid "Moved %{monster}"
-#~ msgstr "Преместено %{monster}"
+"Проверете най-новата версия на играта на\n"
+"https://github.com/ihhub/\n"
+"fheroes2/releases"
```
Comment (zenseii) [inline files/lang/bg.po:9083]: Hi @Effektus.

Please add a new line after "fheroes2/releases" like the warning is talking about here. This is the follow the standard of this project.

**Review by zenseii — APPROVED** [review]

**Comment by ihhub** [comment]
@Effektus , huge thanks for updating this translation!

**Pull request merged**